src/styles/bloks/product-card.scss

```diff
@@ -0,0 +1,104 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  width: $product-card-width;
+  border: 1px solid $border-color;
+  border-radius: $border-radius;
+
+  &__image {
+    background-image: url('../images/imac.jpeg');
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+    box-sizing: border-box;
+  }
+
+  &__content {
+    margin: 40px 16px 16px;
+    font-family: $main-fonts, sans-serif;
+  }
+}
+
+.product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-text-color;
+  margin: 0;
+}
+
+.product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: $second-text-color;
+  margin: 4px 0 0;
+}
+
+.product-rating {
+  margin: 16px 0 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-text-color;
+    margin: 0;
+  }
+}
+
+.product-price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 24px 0 0;
+
+  &__text {
+    line-height: 18px;
+    font-weight: 400;
+    font-size: 12px;
+    color: $second-text-color;
+    margin: 0;
+  }
+
+  &__value {
+    line-height: 18px;
+    font-weight: 700;
+    font-size: 16px;
+    color: $main-text-color;
+    margin: 0;
+  }
+}
+
+.buy-button {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: $button-color;
```

Comment:
Ensure that the variable `$button-color` is defined in your SCSS files. Without this definition, the SCSS will not compile correctly.